### ISSUE TITLE
planning an update?

### ISSUE THREAD

**Issue by linux-paul** [rudetrooper/Octoprint-Chituboard#40]
Will there be an update for ctbV4 and ctbV4_encrypted support soon? I have started working on an implementation of ctbV4_encrypted myself. :)

**Comment by rudetrooper** [comment]
Yeah, I have it mostly finished. Just need to finish some final bits before release, some life events got in the way. Should be released with a week or 2.

**Comment by linux-paul** [comment]
Pretty cool, Thank You, :)

**Comment by DonChooluvit** [comment]
That is AWESOME!! Looking forward to it. Again, I will happily test on a Saturn 2

**Comment by Millerboy3** [comment]
> Yeah, I have it mostly finished. Just need to finish some final bits before release, some life events got in the way. Should be released with a week or 2.

thats amazing news!

**Comment by bvaerewyck** [comment]
You are a rockstar!  I have two orange pi pc's with the OTG ready.   I just need to get the ventilation system ready.

**Comment by SnowFarmsEngineering** [comment]
Great! I am looking forward to seeing if I can get my Mars 3 working with your plugin. Thank you!

**Comment by mbudnek** [comment]
@rudetrooper Any update on this?  I'd love to start using this plugin with my Saturn 2.

**Comment by ChristophCaina** [comment]
Hi :)
Would be really nice to know, if there's an update about the new ctbv4 format :)
I was just ready to start with Octoprint for my Mars2Pro - but then I've heard, that the new ctb4 are not supported...
